gf-guide/prioritization.md

```diff
@@ -0,0 +1,94 @@
+<link href="style.css" rel="stylesheet">
+
+<a href="./index"><button class="button button-i">&larr; GF Guide Index</button></a>
+
+# Prioritization of Adding & Upgrading Fonts to Google Fonts
+{:.no_toc}
+
+<div class="context-reading">
+  
+  Background reading:  
+     <mark class="blue">start</mark> **[Adding & Upgrading Fonts to Google Fonts](./onboarding)**</a>
+     
+</div>
+
+## Table of contents
+{:.no_toc}
+* TOC goes here
+{:toc}
+
+## Priority Definitions
+
+### P0: Top Priority
+
+A large Google Product is blocked from achieving some goal, or soon to be blocked?
+
+This will get a "P0" tag, and is our top priority.
+These are (and should be) very rare on an annual basis.
+
+Target schedule: ASAP (As Soon As Possible, to the exclusion of other tasks/projects.)
+
+### P1: High Priority
+
+A large Google Product wants to use this project, but won't be blocked (so, the Google Fonts team has some time)?
+
+A significant Google customer (e.g. a large organization using Workspace) is blocked, soon to be blocked, or is unhappy enough to consider leaving?
+
+A negative PR event could result, such as missing a major industry deadline (e.g. an upcoming Unicode release)?
+
+This will get a "P1" tag, and is our top priority given no active P0 projects.
+These are (and should be) rare per quarter, but are expect a few times a year.
+
+Target schedule: Defined timeline from the customer.
+
+### P2: Important Priority
+
+A small Google Product, or paying customer, or a non-paying significant customer/user, needs it?
+
+A minor PR event could be negatively impacted?
+
+An existing high usage family has a bug, and the fix will improve pages/documents already using it in a meaningful way?
+
+This will get a "P2" tag, and is given priority to be completed within the planned quarter, given no pending P0 or P1 projects.
+These are expected to make up a small share of activity per quarter.
+
+Target schedule: Quarterley, but can slip.
+
+### P3: General Priority
+
+A nice project that [@davelab6](https://github.com/davelab6), [@chrissimpkins](https://github.com/chrissimpkins) or others in the Google Fonts team would like to ship sooner than later?
```

Comment:
IMO just indicate "Google Fonts team" and remove all individual names.  This keeps it evergreen.  You never know... :)